solidity/contracts/bridge/Bridge.sol

```diff
@@ -138,12 +197,69 @@ contract Bridge is Ownable {
     ///         execute a sweep.
     mapping(uint256 => DepositInfo) public deposits;
 
-    /// @notice Maps the wallet public key hash (computed using HASH160 opcode)
-    ///         to the latest wallet's main UTXO computed as
+    /// @notice Maps the 20-byte wallet public key hash (computed using HASH160
+    ///         opcode) to the latest wallet's main UTXO computed as
     ///         keccak256(txHash | txOutputIndex | txOutputValue). The `tx`
     ///         prefix refers to the transaction which created that main UTXO.
+    ///         The txHash is LE bytes32, txOutputIndex an uint32, and
+    ///         txOutputValue an uint64 value.
     mapping(bytes20 => bytes32) public mainUtxos;
 
+    /// @notice Maps the 20-byte wallet public key hash (computed using HASH160
+    ///         opcode) to the total redeemable value of pending redemption
+    ///         requests targeting that wallet. Redeemable value of each request
+    ///         is its requested amount reduced by the treasury fee at the
+    ///         moment of request creation.
+    mapping(bytes20 => uint256) public pendingRedemptionRequestsValue;
```

Comment:
Does it have to be uint256? In `DepositInfo` amount is `uint64` and I think 64 unsigned int bits should be enough to represent max BTC supply in sats. See my other comment about `Wallet` struct. If we put value and wallet state together we can save some gas given how structs are stored in the memory.

Reply:
See a1b472e2